### PR TITLE
games-board/pychess: bump to version 0.99.4

### PR DESCRIPTION
**Pull request by atomsymbol**
Please merge. Thanks.

Closes: https://bugs.gentoo.org/675128
Signed-off-by: Jan Ziak <0xe2.0x9a.0x9b@gmail.com>

### CODE REVIEW

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{5,6} )
+PYTHON_REQ_USE="sqlite"
+
+inherit gnome2-utils distutils-r1 xdg-utils
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: make `distutils-r1` the last inherit

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	x11-libs/pango
+	x11-themes/adwaita-icon-theme
+	gstreamer? (
+		dev-python/gst-python:1.0
+		media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0 )
+"
+RDEPEND=${DEPEND}
+
+python_install() {
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: this needs to go before `pkg_preinst`

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	# bug 487706
+	sed -i \
+		-e "s/@PYTHON@/${EPYTHON}/" \
+		"${ED%/}/$(python_get_sitedir)"/${PN}/Players/engineNest.py || die
+}
+
+python_install_all() {
+	DOCS="AUTHORS README.md" \
+		distutils-r1_python_install_all
+}
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: remove completely

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+src_prepare() {
+	distutils-r1_src_prepare
+}
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: remove completely

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+src_compile() {
+	distutils-r1_src_compile
+}
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: remove completely

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+src_install() {
+	distutils-r1_src_install
+}
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: remove completely

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+"
+RDEPEND=${DEPEND}
+
+python_install() {
+	distutils-r1_python_install
+
+	# bug 487706
+	sed -i \
+		-e "s/@PYTHON@/${EPYTHON}/" \
+		"${ED%/}/$(python_get_sitedir)"/${PN}/Players/engineNest.py || die
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: is this really necessary I'm wondering? Shebangs aren't normally needed for modules, only for scripts

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+	distutils-r1_src_compile
+}
+
+src_install() {
+	distutils-r1_src_install
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
+}
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: not needed anymore, remove

File: games-board/pychess/pychess-0.99.4.ebuild (hunk 7 above)
Comment (atomsymbol) [reply games-board/pychess/pychess-0.99.4.ebuild]: It was a leftover from older versions of the package, it's not needed anymore.

File: games-board/pychess/pychess-0.99.4.ebuild [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+DEPEND="
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	gnome-base/librsvg:2
+	x11-libs/gtksourceview:3.0
+	x11-libs/pango
+	x11-themes/adwaita-icon-theme
+	gstreamer? (
+		dev-python/gst-python:1.0
```
Comment (SoapGentoo) [inline games-board/pychess/pychess-0.99.4.ebuild]: doesn't this also need `[${PYTHON_USEDEP}]`?

Comment (SoapGentoo) [reply games-board/pychess/pychess-0.99.4.ebuild]: also, have you tried `EAPI=7`?

Comment (atomsymbol) [reply games-board/pychess/pychess-0.99.4.ebuild]: Added both. Thanks.

**Comment by SoapGentoo** [comment]
@atomsymbol thanks, made some minor touchups, have a look (why did you remove `pkg_postrm`?)

**Comment by atomsymbol** [comment]
> @atomsymbol thanks, made some minor touchups, have a look (why did you remove `pkg_postrm`?)

I do Gentoo pull requests only occasionally. It would be nicer for `xdg_*_update` do be invoked automatically if the package modifies (add to or removes from) XDG directories.

Thank you for your suggestions on improving the ebuild.

**Comment by SoapGentoo** [comment]
There are plans for automatic XDG hooks, that will require a PMS update either way.

**Pull request closed without merging**